src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 	 		
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	//*[@id="dependencies-products-change_-3"]/tbody/tr[1]/td[1]/a/strong
+	 private WebElement clickDependentProduct6;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct6() throws IOException {
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 1);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 	 		
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr/td[1]/a/strong
+	 private WebElement clickDependentProduct7;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct7() throws IOException {
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 2);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 	 		
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 private WebElement clickDependentProduct8;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct8() throws IOException {
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 0);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 
+	 	
+	 	
+	 
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 private WebElement clickDependentProduct9;
+
+	 	 		/**
+	 	 		 * Method to click on dependent product.
+	 	 		 * 
+	 	 		 * @throws IOException
+	 	 		 */
+	 public void clickDependentProduct9() throws IOException {
+	 	 	CreateOrderPage sp = new CreateOrderPage(driver);
+	 	 	log.info("click on dependenct product.");
+	 	 	String DependentProduct = BasePage.getCellData(xlsxName, sheetName, 7, 1);
+	 	 	driver.findElement(By.xpath("//a[not(@href)]/strong[text()='" + DependentProduct + "']")).click();
+	 	 		}
+	 
+	 //@FindBy(how=How.XPATH, using="//*[@id="dependencies-products-change_-3"]/tbody/tr[3]/td[1]/a/strong
+	 	private WebElement clickDependentProduct10;
+
+		 	 		/**
+		 	 		 * Method to click on dependent product.
+		 	 		 * 
+		 	 		 * @throws IOException
```

Comment:
@mayankchhajer Please remove the unused WebElements.

Reply:
Done